questions-answers-application/Form1.vb

```diff
@@ -0,0 +1,3 @@
+﻿Public Class Question
+
```

Comment:
What is this? You're missing the create question implementation. For help, see: https://github.com/MyTeamName/questions-answers-app/blob/14e4df438ec644fe6e5c70e15c53a49d468bdd0d/questions-answers-application/CreateQuestion.vb#L4